lib/evm/precompiles/index.ts

```diff
@@ -27,4 +28,4 @@ function getPrecompile(address: string): PrecompileFunc {
   return precompiles[address]
 }
 
-export { precompiles, getPrecompile, PrecompileFunc, PrecompileInput, PrecompileResult }
+export { precompiles, getPrecompile, PrecompileFunc, PrecompileInput, ExecResult }
```

Comment:
Not sure why we need to re-export `ExecResult` here?

Reply:
Good catch! I removed it.